infrastructure/main.tf

```diff
@@ -86,7 +41,7 @@ module "db-rd-judicial-ref-v16" {
   admin_user_object_id = var.jenkins_AAD_objectId
   business_area        = "cft"
   common_tags          = var.common_tags
-  component            = var.component-v16
+  component            = var.component
```

Comment:
if we change this, new db is going to be created?

Reply:
not sure it is being used . but reverting as its a big risk